CHANGES.rst

```diff
@@ -0,0 +1,53 @@
+####################
+crate_ruby changelog
+####################
+
+
+in progress
+===========
+Modernize repository and testing environment
+
+- Use more recent CrateDB version 4.3.2
+- Replace Travis CI by GitHub Actions
+- Add testing for more recent Ruby 2.7 and 3.0
+- Drop testing for older Ruby 2.0, 2.1 and 2.2
+- Make testing work on macOS
+- Fix executable permissions on "java" bundled with recent CrateDB
+- Enable CI testing for macOS
+- Update test framework to rspec 3.10
+- Add Gemfile.lock to repository
+- Rename "history.txt" to "CHANGES.rst"
+- Update development documentation
+
+
+0.2.0
+=====
+- Add SSL support
```

Comment:
I think each section title should be followed by an empty line.

Reply:
Right, the documentation standard is to have two empty lines after each section and one between the header and the section (see [here](https://github.com/crate/crate-docs/blob/main/style/rst.rst))